recipes/libignition-msgs/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libignition-msgs1" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/ignitionrobotics/ign-msgs/get/ignition-msgs_1.0.0.tar.gz
+    sha256: c37e4f6d407ef6d76158a27224705b9c6c12492b8f8c3bf2ece8d0a3431069ca
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage('libignition-msgs1', max_pin='x.x') }}
```

Comment:
you can re-use `name` here, e.g. `{{ pin_subpackage(name, max_pin='x.x') }}` (if you want to do the trick from above, we need to `set` that variable and then use it. maybe

```
{% set base_name = 'libignition-msgs' %}
{% set name = base_name + version.split('.')[0] %}
...
```

although I don't know if the syntax above actually works.

Reply:
Good suggestion. I will be in favor of `name` to eliminate the hard-coded package names, and the example you showed here seems to work for me.